package co.g2academy.exceptionhanlding;

import org.junit.Test;

public class TestTryCatch1 {
    @Test
    public void testAritmaticException() {
        int data = 50/0;

    }
}
